fix(service): update vercel api paths

ndom91 opened the pull request:
We work with the Vercel API at work as a partner of theirs. They move fast and often have many versions in flight at once haha.

The paths you had configured weren't working anymore. These seemed to do the trick for me :+1:

f3l1x commented on services/vercel.js:
```diff
@@ -2,11 +2,11 @@ const axios = require("axios");
     return res.data;
   },
   async getDeployment(id) {
-    const res = await this._getClient().get(`/v11/now/deployments/${id}?projectId=${this._getApiProjectId()}&teamId=${this._getApiTeamId()}`);
```
Current version of this endpoint is v11, according to https://vercel.com/docs/api#endpoints/deployments/get-a-single-deployment.

Did you try it with your change and v11?

ndom91 replied:
Yeah v11 didn't return anything. I checked my dev tools while browsing to the appropriate place in their webapp and there were multiple requests actually with the deploymentId, `v5` and `v7` I believe. One of them didn't work here so I went with the other that did return all the necessary data in the correct shape.

ndom91 replied:
Just double checked, seems to be `v10` and `v13` they're using atm.

f3l1x commented:
Thx. Seems good to me drop teamId. But I see v11 of `Get a Single Deployment` in official vercel docs.

ndom91 commented:
Okay I've removed the `teamId` stuff from a few more places then, and also updated the readme a bit. 

I'm relatively new to strapi and this was the first plugin i'd tried to use. I wasn't aware at first that you can create all those directories under `/admin/src/..` in order to customize the frontend yourself. Took me a while to figure out haha so I added a little not here in the readme for future newbies.

johannessteu commented:
Could this be merged @f3l1x  ?

f3l1x commented:
Bumped v0.0.5. Thank you
